extjsdk/README.md

```diff
@@ -154,6 +154,14 @@ Query errors are sent when a Query cannot be completed successfully. To send a Q
 *	The parameters are an Object array of that will be translated into strings and inserted into the message template
     based anywhere that `{<array index>}` is located.
 
+#### Sending Messages when Vantiq connection drops
+Over the lifetime of a connector, it is possible for the connection between itself and the Vantiq server to drop or get 
```

Comment:
might want to tweak the wording here so that it doesn't sound like a reconnect drops or restarts the connection.  I know what you mean, but it sounds like we're dropping the websocket connection here which we're not.